Review and fix rootcheck API integration test

|Component|
|---|
| API Integration Tests |

This issue is part of #10918 .

The `rootcheck` API integration test is failing in the test case "Check if data has been deleted in agent 001". In this test case, we expect to have `rootcheck` data deleted from agent 001, as it is done after a request to `DELETE /rootcheck` endpoint. 

Test failing in Wazuh `master` branch.

<details>

<summary>Output</summary>

```
============================= test session starts ==============================
platform linux -- Python 3.8.5, pytest-5.4.3, py-1.9.0, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.8.5', 'Platform': 'Linux-5.4.0-1029-aws-x86_64-with-glibc2.29', 'Packages': {'pytest': '5.4.3', 'py': '1.9.0', 'pluggy': '0.13.1'}, 'Plugins': {'tavern': '1.0.0', 'metadata': '1.10.0', 'html': '2.0.1'}}
rootdir: /tmp/Test_integration_endpoints_B1707_test_files/api/test/integration, inifile: pytest.ini
plugins: tavern-1.0.0, metadata-1.10.0, html-2.0.1
collecting ... collected 4 items

test_rootcheck_endpoints.tavern.yaml::GET /rootcheck/001 PASSED          [ 25%]
test_rootcheck_endpoints.tavern.yaml::GET /rootcheck/001/last_scan PASSED [ 50%]
test_rootcheck_endpoints.tavern.yaml::PUT /rootcheck PASSED              [ 75%]
test_rootcheck_endpoints.tavern.yaml::DELETE /rootcheck/{agent_id} FAILED [100%]

=================================== FAILURES ===================================
_ /tmp/Test_integration_endpoints_B1707_test_files/api/test/integration/test_rootcheck_endpoints.tavern.yaml::DELETE /rootcheck/{agent_id} _
Format variables:
  protocol:s = 'https'
  host:s = 'localhost'
  port:d = '55000'
  test_login_token = 'eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjM5MTc3NTIwLCJleHAiOjE2MzkxNzg0MjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbMV0sInJiYWNfbW9kZSI6IndoaXRlIn0.AY9GiuJFEJhUlALFCPJdgvSmHdrfQqjWBavAAXS8FjTsOOTm0ZMryhc6IrannoFbXGSQ5OqwVoQRwBMgyVBhX7acALb5De-v4Pd7j9XZHOhV1WpLBGraaj9pSUeTEDIpCl6Pw37dCbEq9c6nkd4ITHRLhykRdhi2aY619IxEC9kfqq2w'

Source test stage (line 575):
  - name: Check if data has been deleted in agent 001
    request:
      verify: False
      method: GET
      url: "{protocol:s}://{host:s}:{port:d}/rootcheck/001"
      headers:
        Authorization: "Bearer {test_login_token}"
    response:
      status_code: 200
      json:
        error: 0
        data:
          affected_items: []
          failed_items: []
          total_affected_items: 0
          total_failed_items: 0
        message: !anystr

Formatted stage:
  name: Check if data has been deleted in agent 001
  request:
    headers:
      Authorization: Bearer eyJhbGciOiJFUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ3YXp1aCIsImF1ZCI6IldhenVoIEFQSSBSRVNUIiwibmJmIjoxNjM5MTc3NTIwLCJleHAiOjE2MzkxNzg0MjAsInN1YiI6InRlc3RpbmciLCJydW5fYXMiOmZhbHNlLCJyYmFjX3JvbGVzIjpbMV0sInJiYWNfbW9kZSI6IndoaXRlIn0.AY9GiuJFEJhUlALFCPJdgvSmHdrfQqjWBavAAXS8FjTsOOTm0ZMryhc6IrannoFbXGSQ5OqwVoQRwBMgyVBhX7acALb5De-v4Pd7j9XZHOhV1WpLBGraaj9pSUeTEDIpCl6Pw37dCbEq9c6nkd4ITHRLhykRdhi2aY619IxEC9kfqq2w
    method: GET
    url: 'https://localhost:55000/rootcheck/001'
    verify: false
  response:
    json:
      data:
        affected_items: []
        failed_items: []
        total_affected_items: 0
        total_failed_items: 0
      error: 0
      message: !anystr ''
    status_code: 200

Errors:
E   tavern.util.exceptions.TestFailError: Test 'Check if data has been deleted in agent 001' failed:
    - Key mismatch: (expected["data"]["total_affected_items"] = '0' (type = <class 'int'>), actual["data"]["total_affected_items"] = '5' (type = <class 'int'>))
------------------------------ Captured log call -------------------------------
WARNING  tavern._plugins.rest.response:response.py:29 Unexpected status code '200'
WARNING  tavern._plugins.rest.response:response.py:29 Unexpected status code '200'
WARNING  tavern._plugins.rest.response:response.py:29 Unexpected status code '200'
ERROR    tavern.response.base:base.py:41 Key mismatch: (expected["data"]["total_affected_items"] = '0' (type = <class 'int'>), actual["data"]["total_affected_items"] = '5' (type = <class 'int'>))
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert {'data': {'affected_items': [{'cis': '1.4 Debian Linux',\n                              'date_first': '2021-12-10T23:05:22Z',\n                              'date_last': '2021-12-10T23:05:22Z',\n                              'log': 'System Audit: CIS - Debian Linux - 1.4 - '\n                                     'Robust partition scheme - /opt is not on '\n                                     'its own partition {CIS: 1.4 Debian '\n                                     'Linux}. File: /opt. Reference: '\n                                     'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                                     '.',\n                              'status': 'outstanding'},\n                             {'cis': '1.4 Debian Linux',\n                              'date_first': '2021-12-10T23:05:22Z',\n                              'date_last': '2021-12-10T23:05:22Z',\n                              'log': 'System Audit: CIS - Debian Linux - 1.4 - '\n                                     'Robust partition scheme - /tmp is not on '\n                                     'its own partition {CIS: 1.4 Debian '\n                                     'Linux}. File: /etc/fstab. Reference: '\n                                     'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                                     '.',\n                              'status': 'outstanding'},\n                             {'cis': '1.4 Debian Linux',\n                              'date_first': '2021-12-10T23:05:22Z',\n                              'date_last': '2021-12-10T23:05:22Z',\n                              'log': 'System Audit: CIS - Debian Linux - 1.4 - '\n                                     'Robust partition scheme - /var is not on '\n                                     'its own partition {CIS: 1.4 Debian '\n                                     'Linux}. File: /etc/fstab. Reference: '\n                                     'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                                     '.',\n                              'status': 'outstanding'},\n                             {'cis': '5.2 Debian Linux',\n                              'date_first': '2021-12-10T23:05:22Z',\n                              'date_last': '2021-12-10T23:05:22Z',\n                              'log': 'System Audit: CIS - Debian Linux - 5.2 - '\n                                     'Network parameters - IP Forwarding '\n                                     'enabled {CIS: 5.2 Debian Linux}. File: '\n                                     '/proc/sys/net/ipv4/ip_forward. '\n                                     'Reference: '\n                                     'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                                     '.',\n                              'status': 'outstanding'},\n                             {'date_first': '2021-12-10T23:05:22Z',\n                              'date_last': '2021-12-10T23:05:22Z',\n                              'log': 'System Audit: CIS - Testing against the '\n                                     'CIS Debian Linux Benchmark v1.0. File: '\n                                     '/etc/debian_version. Reference: '\n                                     'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                                     '.',\n                              'status': 'outstanding'}],\n          'failed_items': [],\n          'total_affected_items': 5,\n          'total_failed_items': 0},\n 'error': 0,\n 'message': 'All selected rootcheck information was returned'} == {'data': {'affected_items': [],\n          'failed_items': [],\n          'total_affected_items': 0,\n          'total_failed_items': 0},\n 'error': 0,\n 'message': <tavern.util.loader.StrSentinel object at 0x7f949e2e3190>}
  Common items:
  {'error': 0}
  Differing items:
  {'message': 'All selected rootcheck information was returned'} != {'message': <tavern.util.loader.StrSentinel object at 0x7f949e2e3190>}
  {'data': {'affected_items': [{'cis': '1.4 Debian Linux', 'date_first': '2021-12-10T23:05:22Z', 'date_last': '2021-12-1...chmark_v1.0.pdf .', 'status': 'outstanding'}], 'failed_items': [], 'total_affected_items': 5, 'total_failed_items': 0}} != {'data': {'affected_items': [], 'failed_items': [], 'total_affected_items': 0, 'total_failed_items': 0}}
  Full diff:
    {
  -  'data': {'affected_items': [],
  +  'data': {'affected_items': [{'cis': '1.4 Debian Linux',
  +                               'date_first': '2021-12-10T23:05:22Z',
  +                               'date_last': '2021-12-10T23:05:22Z',
  +                               'log': 'System Audit: CIS - Debian Linux - 1.4 - '
  +                                      'Robust partition scheme - /opt is not on '
  +                                      'its own partition {CIS: 1.4 Debian '
  +                                      'Linux}. File: /opt. Reference: '
  +                                      'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                                      '.',
  +                               'status': 'outstanding'},
  +                              {'cis': '1.4 Debian Linux',
  +                               'date_first': '2021-12-10T23:05:22Z',
  +                               'date_last': '2021-12-10T23:05:22Z',
  +                               'log': 'System Audit: CIS - Debian Linux - 1.4 - '
  +                                      'Robust partition scheme - /tmp is not on '
  +                                      'its own partition {CIS: 1.4 Debian '
  +                                      'Linux}. File: /etc/fstab. Reference: '
  +                                      'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                                      '.',
  +                               'status': 'outstanding'},
  +                              {'cis': '1.4 Debian Linux',
  +                               'date_first': '2021-12-10T23:05:22Z',
  +                               'date_last': '2021-12-10T23:05:22Z',
  +                               'log': 'System Audit: CIS - Debian Linux - 1.4 - '
  +                                      'Robust partition scheme - /var is not on '
  +                                      'its own partition {CIS: 1.4 Debian '
  +                                      'Linux}. File: /etc/fstab. Reference: '
  +                                      'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                                      '.',
  +                               'status': 'outstanding'},
  +                              {'cis': '5.2 Debian Linux',
  +                               'date_first': '2021-12-10T23:05:22Z',
  +                               'date_last': '2021-12-10T23:05:22Z',
  +                               'log': 'System Audit: CIS - Debian Linux - 5.2 - '
  +                                      'Network parameters - IP Forwarding '
  +                                      'enabled {CIS: 5.2 Debian Linux}. File: '
  +                                      '/proc/sys/net/ipv4/ip_forward. '
  +                                      'Reference: '
  +                                      'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                                      '.',
  +                               'status': 'outstanding'},
  +                              {'date_first': '2021-12-10T23:05:22Z',
  +                               'date_last': '2021-12-10T23:05:22Z',
  +                               'log': 'System Audit: CIS - Testing against the '
  +                                      'CIS Debian Linux Benchmark v1.0. File: '
  +                                      '/etc/debian_version. Reference: '
  +                                      'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                                      '.',
  +                               'status': 'outstanding'}],
              'failed_items': [],
  -           'total_affected_items': 0,
  ?                                   ^
  +           'total_affected_items': 5,
  ?                                   ^
              'total_failed_items': 0},
     'error': 0,
  -  'message': <tavern.util.loader.StrSentinel object at 0x7f949e2e3190>,
  +  'message': 'All selected rootcheck information was returned',
    }

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert {'affected_items': [{'cis': '1.4 Debian Linux',\n                     'date_first': '2021-12-10T23:05:22Z',\n                     'date_last': '2021-12-10T23:05:22Z',\n                     'log': 'System Audit: CIS - Debian Linux - 1.4 - Robust '\n                            'partition scheme - /opt is not on its own '\n                            'partition {CIS: 1.4 Debian Linux}. File: /opt. '\n                            'Reference: '\n                            'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                            '.',\n                     'status': 'outstanding'},\n                    {'cis': '1.4 Debian Linux',\n                     'date_first': '2021-12-10T23:05:22Z',\n                     'date_last': '2021-12-10T23:05:22Z',\n                     'log': 'System Audit: CIS - Debian Linux - 1.4 - Robust '\n                            'partition scheme - /tmp is not on its own '\n                            'partition {CIS: 1.4 Debian Linux}. File: '\n                            '/etc/fstab. Reference: '\n                            'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                            '.',\n                     'status': 'outstanding'},\n                    {'cis': '1.4 Debian Linux',\n                     'date_first': '2021-12-10T23:05:22Z',\n                     'date_last': '2021-12-10T23:05:22Z',\n                     'log': 'System Audit: CIS - Debian Linux - 1.4 - Robust '\n                            'partition scheme - /var is not on its own '\n                            'partition {CIS: 1.4 Debian Linux}. File: '\n                            '/etc/fstab. Reference: '\n                            'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                            '.',\n                     'status': 'outstanding'},\n                    {'cis': '5.2 Debian Linux',\n                     'date_first': '2021-12-10T23:05:22Z',\n                     'date_last': '2021-12-10T23:05:22Z',\n                     'log': 'System Audit: CIS - Debian Linux - 5.2 - Network '\n                            'parameters - IP Forwarding enabled {CIS: 5.2 '\n                            'Debian Linux}. File: '\n                            '/proc/sys/net/ipv4/ip_forward. Reference: '\n                            'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                            '.',\n                     'status': 'outstanding'},\n                    {'date_first': '2021-12-10T23:05:22Z',\n                     'date_last': '2021-12-10T23:05:22Z',\n                     'log': 'System Audit: CIS - Testing against the CIS '\n                            'Debian Linux Benchmark v1.0. File: '\n                            '/etc/debian_version. Reference: '\n                            'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '\n                            '.',\n                     'status': 'outstanding'}],\n 'failed_items': [],\n 'total_affected_items': 5,\n 'total_failed_items': 0} == {'affected_items': [],\n 'failed_items': [],\n 'total_affected_items': 0,\n 'total_failed_items': 0}
  Common items:
  {'failed_items': [], 'total_failed_items': 0}
  Differing items:
  {'affected_items': [{'cis': '1.4 Debian Linux', 'date_first': '2021-12-10T23:05:22Z', 'date_last': '2021-12-10T23:05:2... Reference: https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf .', 'status': 'outstanding'}]} != {'affected_items': []}
  {'total_affected_items': 5} != {'total_affected_items': 0}
  Full diff:
    {
  -  'affected_items': [],
  +  'affected_items': [{'cis': '1.4 Debian Linux',
  +                      'date_first': '2021-12-10T23:05:22Z',
  +                      'date_last': '2021-12-10T23:05:22Z',
  +                      'log': 'System Audit: CIS - Debian Linux - 1.4 - Robust '
  +                             'partition scheme - /opt is not on its own '
  +                             'partition {CIS: 1.4 Debian Linux}. File: /opt. '
  +                             'Reference: '
  +                             'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                             '.',
  +                      'status': 'outstanding'},
  +                     {'cis': '1.4 Debian Linux',
  +                      'date_first': '2021-12-10T23:05:22Z',
  +                      'date_last': '2021-12-10T23:05:22Z',
  +                      'log': 'System Audit: CIS - Debian Linux - 1.4 - Robust '
  +                             'partition scheme - /tmp is not on its own '
  +                             'partition {CIS: 1.4 Debian Linux}. File: '
  +                             '/etc/fstab. Reference: '
  +                             'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                             '.',
  +                      'status': 'outstanding'},
  +                     {'cis': '1.4 Debian Linux',
  +                      'date_first': '2021-12-10T23:05:22Z',
  +                      'date_last': '2021-12-10T23:05:22Z',
  +                      'log': 'System Audit: CIS - Debian Linux - 1.4 - Robust '
  +                             'partition scheme - /var is not on its own '
  +                             'partition {CIS: 1.4 Debian Linux}. File: '
  +                             '/etc/fstab. Reference: '
  +                             'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                             '.',
  +                      'status': 'outstanding'},
  +                     {'cis': '5.2 Debian Linux',
  +                      'date_first': '2021-12-10T23:05:22Z',
  +                      'date_last': '2021-12-10T23:05:22Z',
  +                      'log': 'System Audit: CIS - Debian Linux - 5.2 - Network '
  +                             'parameters - IP Forwarding enabled {CIS: 5.2 '
  +                             'Debian Linux}. File: '
  +                             '/proc/sys/net/ipv4/ip_forward. Reference: '
  +                             'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                             '.',
  +                      'status': 'outstanding'},
  +                     {'date_first': '2021-12-10T23:05:22Z',
  +                      'date_last': '2021-12-10T23:05:22Z',
  +                      'log': 'System Audit: CIS - Testing against the CIS '
  +                             'Debian Linux Benchmark v1.0. File: '
  +                             '/etc/debian_version. Reference: '
  +                             'https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Benchmark_v1.0.pdf '
  +                             '.',
  +                      'status': 'outstanding'}],
     'failed_items': [],
  -  'total_affected_items': 0,
  ?                          ^
  +  'total_affected_items': 5,
  ?                          ^
     'total_failed_items': 0,
    }

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/tavern/util/dict_util.py", line 406, in check_keys_match_recursive
    assert actual_val == expected_val
AssertionError: assert 5 == 0
  +5
  -0

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/tavern/response/base.py", line 88, in recurse_check_key_match
    check_keys_match_recursive(expected_block, block, [], strict)
  File "/usr/local/lib/python3.8/dist-packages/tavern/util/dict_util.py", line 462, in check_keys_match_recursive
    check_keys_match_recursive(
  File "/usr/local/lib/python3.8/dist-packages/tavern/util/dict_util.py", line 462, in check_keys_match_recursive
    check_keys_match_recursive(
  File "/usr/local/lib/python3.8/dist-packages/tavern/util/dict_util.py", line 546, in check_keys_match_recursive
    raise exceptions.KeyMismatchError(
tavern.util.exceptions.KeyMismatchError: Key mismatch: (expected["data"]["total_affected_items"] = '0' (type = <class 'int'>), actual["data"]["total_affected_items"] = '5' (type = <class 'int'>))
=============================== warnings summary ===============================
/usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/hooks.py:83
  /usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/hooks.py:83: PytestDeprecationWarning: direct construction of YamlFile has been deprecated, please use YamlFile.from_parent
    return YamlFile(path, parent)

/usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:233
/usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:233
/usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:233
/usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:233
  /usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:233: PytestDeprecationWarning: direct construction of YamlItem has been deprecated, please use YamlItem.from_parent
    item = YamlItem(test_spec["test_name"], self, test_spec, self.fspath)

/usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:61
  /usr/local/lib/python3.8/dist-packages/tavern/testutils/pytesthook/file.py:61: PytestUnknownMarkWarning: Unknown pytest.mark.base_tests - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    pytest_marks.append(getattr(pytest.mark, m))

-- Docs: https://docs.pytest.org/en/latest/warnings.html
- generated html file: file:///tmp/Test_integration_endpoints_B1707_test_files/api/test/integration/Test_integration_endpoints_B1707_test_rootcheck_endpoints.html -
=========================== short test summary info ============================
FAILED test_rootcheck_endpoints.tavern.yaml::DELETE /rootcheck/{agent_id}
============= 1 failed, 3 passed, 6 warnings in 351.71s (0:05:51) ==============
```

</details>


We should review and fix this API integration test.


## Checks

### wazuh/wazuh
- Unit tests without failures. Updated and/or expanded if there are new functions/methods/outputs:
  - [ ] Cluster (`framework/wazuh/core/cluster/tests/` & `framework/wazuh/core/cluster/dapi/tests/`)
  - [ ] Core (`framework/wazuh/core/tests/`)
  - [ ] SDK (`framework/wazuh/tests/`)
  - [ ] RBAC (`framework/wazuh/rbac/tests/`)
  - [ ] API (`api/api/tests/`)
- API tavern integration tests without failures. Updated and/or expanded if needed (`api/test/integration/`):
  - [ ] Affected tests 
  - [ ] Affected RBAC (black and white) tests
- [ ] Review integration test mapping using the script (`api/test/integration/mapping/integration_test_api_endpoints.json`)
- [ ] Review of spec.yaml examples and schemas (`api/api/spec/spec.yaml`)
- [ ] Review exceptions remediation when any endpoint path changes or is removed (`framework/wazuh/core/exception.py`)
- [ ] Changelog (`CHANGELOG.md`)
<!-- If changes are made to any of the following components, uncomment the corresponding line 
- [ ] **Integration tests** without failures for API configuration (`/wazuh-qa/tests/integration/test_api/test_config/`)
- [ ] **System tests** for agent enrollment process (`/wazuh-qa/tests/system/test_cluster/test_agent_enrollment`)
- [ ] **System tests** for agent info sync process in cluster (`/wazuh-qa/tests/system/test_cluster/test_agent_info_sync`)
- [ ] **System tests** for agent key polling (`/wazuh-qa/tests/system/test_cluster/test_agent_key_polling`)
- [ ] **System tests** for JWT invalidation (`/wazuh-qa/tests/system/test_jwt_invalidation`)
-->

### wazuh/wazuh-documentation
- [ ] Migration from 3.X for changed endpoints (`source/user-manual/api/equivalence.rst`)
- [ ] Update RBAC reference with new/modified actions/resources/relationships (`source/user-manual/api/rbac/reference.rst`)

--- a/api/test/integration/test_rootcheck_endpoints.tavern.yaml
+++ b/api/test/integration/test_rootcheck_endpoints.tavern.yaml
@@ -471,8 +471,8 @@ test_name: PUT /rootcheck
 
 stages:
 
-  # PUT /rootcheck?agents_list=001
-  - name: Run rootcheck scan in agent 001
+  # PUT /rootcheck?agents_list=002
+  - name: Run rootcheck scan in agent 002
     request:
       verify: False
       method: PUT
@@ -480,14 +480,14 @@ stages:
       headers:
         Authorization: "Bearer {test_login_token}"
       params:
-        agents_list: "001"
+        agents_list: "002"
     response:
       status_code: 200
       json:
         error: 0
         data:
           affected_items:
-            - "001"
+            - "002"
           failed_items: []
           total_affected_items: 1
           total_failed_items: 0

